win32ss/user/winsrv/usersrv/harderror.c

```diff
@@ -489,14 +492,12 @@ UserpShowInformationBalloon(
     CopyData.cbData = dwSize;
     CopyData.lpData = pdata;
 
-    dwResult = 0;
-
-    SendMessageTimeoutW(hwnd, WM_COPYDATA, 0, (LPARAM)&CopyData,
-                        SMTO_NORMAL | SMTO_ABORTIFHUNG, 3000, &dwResult);
+    result = SendMessageTimeoutW(hwnd, WM_COPYDATA, 0, (LPARAM)&CopyData,
+                                 SMTO_NORMAL | SMTO_ABORTIFHUNG, 3000, NULL);
```

Comment:
We should also check the value returned by the notification (instead of the NULL in the last parameter).

Reply:
why?

Reply:
The shell may specify whether or not it handles the notification.

Reply:
Yes, with the value it returns.